pkg/resources/statefulsets/prometheus.go

```diff
@@ -129,6 +146,31 @@ func NewForPrometheus(t *miniov1.Tenant, serviceName string) *appsv1.StatefulSet
 	}
 
 	containers := []corev1.Container{prometheusServerContainer(t)}
+
+	// init container per https://github.com/prometheus-operator/prometheus-operator/issues/966#issuecomment-365037713
+	//
+	//  initContainers:
+	//  - name: "init-chown-data"
+	//    image: "busybox"
+	//    # 1000 is the user that prometheus uses.
+	//    command: ["chown", "-R", "1000:2000", /var/prometheus/data]
+	//    volumeMounts:
+	//    - name: prometheus-kube-prometheus-db
+	//      mountPath: /var/prometheus/data
+	initContainers := []corev1.Container{
+		{
+			Name:  "init-chown-data",
```

Comment:
nit: `init-chown-data` may alarm users browsing k8s workloads on tenant's namespace. Prefixing container name with `prometheus` is aimed at clarifying init container's  purpose/scope.
```suggestion
			Name:  "prometheus-init-chown-data",
```

Reply:
done